formats/help/frontend/examples/help_page.json

```diff
@@ -0,0 +1,102 @@
+    "external_related_links": [
```

Comment:
This looks like a feature frontend will need to check/render. It’s probably worth adding a second example that includes some external related links. Is there a live help page that has some?